src/Bundle/Controller/FlashHelper.php

```diff
@@ -89,8 +103,14 @@ private function addFlash(string $type, string $message, array $parameters = [])
             $message = $this->prepareMessage($message, $parameters);
         }
 
+        if ($this->requestStack instanceof SessionInterface) {
```

Comment:
Having afterthought, perhaps we should wrap it into our custom class, that would do this logic?